package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Harmony soccerplayers.app",
   "main": "truffle.js",
   "scripts": {
-    "libInit": "tar -zxf ethereumjs-ethash-dist.tar.gz -C node_modules/@ethereumjs/ethash"
+    "libInit": "tar -zxf ethereumjs-ethash-dist.tar.gz -C node_modules/@ethereumjs/ethash",
```

Comment:
What is this? Where does the .tar.gz package come from and how do relayers and developers verify they are trustworthy?